support-modules/acquisitions-stream/src/main/scala/com/gu/support/acquisitions/AcquisitionsStreamService.scala

```diff
@@ -21,16 +21,16 @@ sealed trait AcquisitionsStreamServiceConfig {
   val streamName: String
 }
 
-case class Ec2OrLocalConfig(val streamName: String, val credentialsProvider: AWSCredentialsProviderChain) extends AcquisitionsStreamServiceConfig
-case class LambdaConfig(val streamName: String) extends AcquisitionsStreamServiceConfig
+case class AcquisitionsStreamEc2OrLocalConfig(val streamName: String, val credentialsProvider: AWSCredentialsProviderChain) extends AcquisitionsStreamServiceConfig
```

Comment:
if module-acquisitions-stream is a new thing, it would be good to use AWS SDK v2 if possible, as we should be moving over aanyway (for snyk reasons and also it's more maaintaained by amazon)